README.md

```diff
@@ -0,0 +1,100 @@
+# KCP Helm Charts
```

Comment:
nit, if you rename with `git mv` I think the diff will only show the rename+changes vs a whole new file?

Reply:
`git mv readme.md README.md` :/ didn't worked

Reply:
In what way didn't work?  I did it [here] (using git 2.39.1 on Fedora) and worked fine :thinking: 

Given the small amount of history on the file it's not a big deal, but would be interesting to understand what the problem is :)

Reply:
same git, same OS... Honestly, it never worked on my git config :D I might need to look into which githook breaks this behaviour on my ned :D